### PR TITLE
Add document about how to specify metadata like binaries, services or…

### PR DESCRIPTION
**Pull request by dholbach**
… config (LP: #1535905).

### CODE REVIEW

**Comment by kyrofa** [comment]
This one needs the LP ref on its own line as well.

File: docs/metadata.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+## Adding binaries
+
+Every part may build a binary that may need to be exposed in the system, with
+that in mind, the way to expose it would be to use the **binaries** keyword
+and declare each binary and its functionality. So, if there were two binaries,
+one called *app1* and another called *app2*, the extract would look like
+
+```yaml
+binaries:
```
Comment (kyrofa) [inline docs/metadata.md]: In master this stuff has moved to 2.0 syntax-- no `binaries` or `services` anymore, only apps.

**Comment by dholbach** [comment]
1.x PR: https://github.com/ubuntu-core/snapcraft/pull/243 - I'll rework the `binaries` and `services` bits for `master`.

File: docs/metadata.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+```yaml
+apps:
+  daemon1:
+    command: bin/app1
+    daemon: simple
+  daemon2:
+    command: bin/app2 --start
+    stop-command: bin/app2 --stop
+    daemon: simple
```
Comment (kyrofa) [inline docs/metadata.md]: Perhaps make this one forking and discuss that capability.

**Comment by kyrofa** [comment]
:+1: from me.

**Comment by dholbach** [comment]
@sergiusens - can we merge this?

**Pull request merged**